libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -23,6 +23,7 @@
 #include <AP_HAL/Util.h>
 #include <DataFlash/DataFlash.h>
 #include <GCS_MAVLink/GCS.h>
+#include <stdio.h>
```

Comment:
Mostly pointed it out, as I don't actually see any new code that uses anything from it, not that worried either way though.